dockerfiles/buildall-common.sh

```diff
@@ -87,7 +112,8 @@ build_image() {
   [[ -z ${buildTag} ]] && buildTag=$BUILD_TAG
 
   if [ "$rebuild" = "1" ] || ! docker pull ${DEST_DOCKER_REGISTRY}/${DOCKER_REPO}/${name}:${label} ; then
-    docker image build -t ${DEST_DOCKER_REGISTRY}/${DEST_DOCKER_REPO}/${name}:${label} \
+    DOCKER_BUILDKIT=1 docker image build \
```

Comment:
question: may be desirable anyway, but curious whether DOCKER_BUILDKIT=1 was required for this change?

Reply:
Buildkit isn't turned on by default at the moment.  On tests with github actions, without explicitly declaring the use of docker's buildkit, it was using the default builder that doesn't have support for mounting secrets into image layers.  This line just ensures it's turned on for our build so we have access to those newer features.